dtdl-parser/src/dtmi.rs

```diff
@@ -71,7 +71,7 @@ impl Dtmi {
 
         let fragment = match iri.fragment() {
             Some(fragment) => String::from(fragment.as_str()),
-            None => String::from(""),
+            None => String::new(),
```

Comment:
maybe have it as a const?

Reply:
Not for this case.  If the iri does not have a fragment, then you just want an empty string and "String::new()" is fine for representing this.